src/Solvers/poisson_eigenvalues.jl

```diff
@@ -5,9 +5,9 @@ Return the eigenvalues satisfying the discrete form of Poisson's equation
 with periodic boundary conditions along the dimension `dim` with `N` grid
 points and domain extent `L`.
 """
-function poisson_eigenvalues(N, L, dim, ::Periodic)
+function poisson_eigenvalues(N, L, dim, ::Periodic, FT)
```

Comment:
probably should make `FT` the first argument as it usually appears in other functions